src/seq/bioseq.jl

```diff
@@ -1100,3 +1098,130 @@ function seqmatrix{A<:Alphabet}(vseq::AbstractVector{BioSequence{A}}, major::Sym
         throw(ArgumentError("major must be :site or :seq"))
     end
 end
+
+"""
+    seqmatrix{A<:Alphabet,T}(vseq::Vector{BioSequence{A}}, major::Symbol, conv::Type{T})
```

Comment:
The type argument should come first (c.f. `convert`, `parse`, `zeros`, etc.). Also, `conv` will not be used and can be safely removed.